deploy/crds/cincinnati.openshift.io_v1alpha1_cincinnati_cr.yaml

```diff
@@ -6,6 +6,13 @@ spec:
   replicas: 1
   registry: "quay.io"
   repository: "openshift-release-dev/ocp-release"
+  # There two ways to specify the source of the Cincinnati graph data.
+  # The first is through a github repository specified by these three
+  # parameters.
   gitHubOrg: "openshift"
   gitHubRepo: "cincinnati-graph-data"
-  branch: "master"
\ No newline at end of file
+  branch: "master"
+  # The second is through an init container specified by graphDataImage.
+  # When specified, graphDataImage takes precedence over gitHubOrg, 
+  # gitHubRepo, and branch. 
+  graphDataImage: "quay.io/rwsu/cincinnati-graph-data-container:latest"
```

Comment:
It would be nice if we could build from this project (ie. `quay.io/openshift/cincinnati-graph-data-container`)

Reply:
For now I think we should avoid any appearance that there might be an "official" build of this init container until we get agreement that as Red Hat we want to produce, ship, and support one. It would be easy for someone to see this feature, see that repo, and just start using it in production because it works. For now this feature is intended for a user who is willing to setup their own build of an init container that snapshots the data at whatever point in time they choose.

I definitely like the idea of having one available for development and testing; I'd just keep it under some less-official-looking namespace. Maybe `quay.io/cincinnati/scarybreakseverythinggraphdata` or similar. :wink: Ok maybe not that, but something clearly unofficial.

Reply:
I think a suitable compromise would be:

1. Not including the `graphDataImage` in the basic CR at all
1. Introducing the `Dockerfile` one could use to build such an image next to relevant documentation about how/why to use it. Probably best to create an issue for the documentation and just add the `Dockerfile` now (maybe in `$PROJECT_ROOT/dev)?